examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_proxy_operator.py

```diff
@@ -119,6 +119,27 @@ def get_dagster_run_status(
         )
         return self.get_valid_graphql_response(response, "runOrError")["status"]
 
+    def get_attribute_from_airflow_context(self, context: Context, attribute: str) -> Any:
+        if attribute not in context or context[attribute] is None:
+            raise Exception(f"Attribute {attribute} not found in context.")
+        return context[attribute]
+
+    def get_airflow_dag_run_id(self, context: Context) -> str:
+        return self.get_attribute_from_airflow_context(context, "dag_run").run_id
+
+    def get_airflow_dag_id(self, context: Context) -> str:
+        return self.get_attribute_from_airflow_context(context, "dag_run").dag_id
+
+    def get_airflow_task_id(self, context: Context) -> str:
+        return self.get_attribute_from_airflow_context(context, "task").task_id
```

Comment:
to constrast from the previous PR, `get` is appropriate here since it is in-memory/cheap